src/main/java/com/github/tomakehurst/wiremock/http/Request.java

```diff
@@ -60,6 +60,10 @@ interface Part {
 
   QueryParameter queryParameter(String key);
 
+  FormParameter formParameter(String key);
```

Comment:
Without default implementation, it breaks binary compatibility

Reply:
incorporated